.github/workflows/tests.yml

```
@@ -291,6 +291,202 @@ jobs:
           name: "${{ matrix.python-version }} ${{ matrix.os }} ${{ matrix.name-suffix }} result images"
           path: ./result_images
 
+  test-cygwin:
+    runs-on: windows-latest
+    name: Python 3.${{ matrix.python-minor-version }} on Cygwin
+    if: |
+      github.event_name == 'workflow_dispatch' ||
+      (
+        github.repository == 'matplotlib/matplotlib' &&
+        !contains(github.event.head_commit.message, '[ci skip]') &&
+        !contains(github.event.head_commit.message, '[skip ci]') &&
+        !contains(github.event.head_commit.message, '[skip github]')
+      )
+    env:
+      SHELLOPTS: igncr
+      CYGWIN_NOWINPATH: 1
+      CHERE_INVOKING: 1
+      TMP: /tmp
+      TEMP: /tmp
+    strategy:
+      matrix:
+        python-minor-version: [8]
```

```suggestion
        python-minor-version: [8, 9]
```

Is this tempting fate too much?